src/main/java/com/kodilla/ecommercee/repositories/ProductRepository.java

```diff
@@ -0,0 +1,22 @@
+package com.kodilla.ecommercee.repositories;
+
+import com.kodilla.ecommercee.domains.Product;
+import org.springframework.data.repository.CrudRepository;
+
+import java.util.List;
+import java.util.Optional;
+
+public interface ProductRepository extends CrudRepository<Product, Long> {
```

Comment:
Nie musisz dodawać metod które są dziedziczone po CrudRepository jeżeli nie chcesz ich modyfikować. Oznacza to, że metody takie jak findAll, save, findById, deleteById nie muszą tutaj być, a i tak można je użyć w obiekcie.

Reply:
A nie wyniknie później z tego że jeżeli będę ich używał to będą musiały być rzutowane typy na Product?

Reply:
Z tego co kojarzę to raczej nie. Dlatego podajesz ten typ w `CrudRepository<Product, Long>`. Przynajmniej tak mi to wcześniej działało.

Reply:
Dobrze, w takim razie to usunę a jeśli byłyby z tym jakieś problemy to wtedy to poprostu się doda :)